CRM/Member/Form/Membership.php

```diff
@@ -925,7 +925,7 @@ private function setOverrideDateValue() {
    * or FALSE otherwise.
    */
   private function convertIsOverrideValue() {
-    $this->_params['is_override'] = CRM_Member_StatusOverrideTypes::isOverridden($this->_params['is_override']);
+    $this->_params['is_override'] = CRM_Member_StatusOverrideTypes::isOverridden($this->_params['is_override'] ?? 0);
```

Comment:
```suggestion
    $this->_params['is_override'] = CRM_Member_StatusOverrideTypes::isOverridden($this->_params['is_override'] ?? CRM_Member_StatusOverrideTypes::NO);
```

In case php 9.x deprecates the number 0 or something (partly joking).